concepts/tuples/about.md

```diff
@@ -1,17 +1,23 @@
 # About
 
 A [tuple][tuple] is an _immutable_ collection of items in _sequence_.
+
+
 Like most collections (_see the built-ins [`list`][list], [`dict`][dict] and [`set`][set]_), `tuples` can hold any (or multiple) data type(s) -- including other `tuples`.
-The elements of a tuple can be iterated over using the `for item in <tuple>` construct.
-If both element index and value are needed, `for index, item in enumerate(<tuple>)` can be used.
+  The elements of a tuple can be iterated over using the `for item in <tuple>` construct.
+  If both element index and value are needed, `for index, item in enumerate(<tuple>)` can be used.
+
+
 Like any [sequence][sequence], elements within `tuples` can be accessed via _bracket notation_ using a `0-based index` number from the left or a `-1-based index` number from the right.
- Tuples can be copied in whole or in part via _slice notation_ or `<tuple>.copy()`, and support all [common sequence operations][common sequence operations].
+  Tuples can be copied in whole or in part via _slice notation_ or `<tuple>.copy()`, and support all [common sequence operations][common sequence operations].
   Being _immutable_, `tuples` **do not** support [mutable sequence operations][mutable sequence operations].
```

Comment:
```suggestion
 Tuples can be copied in whole or in part via _slice notation_ or `<tuple>.copy()`, and support all [common sequence operations][common sequence operations].
Being _immutable_, `tuples` **do not** support [mutable sequence operations][mutable sequence operations].
```

I don't believe the leading spaces are helpful.  This happens on lines 17 and 18 as well.

Reply:
I am going to leave this in right now, but may come back and omit them.   Not having leading spaces is causing the copy (I think) on the site to display without any space after periods.  It's making the copy unreadable.